app/res/values/styles.xml

```diff
@@ -137,4 +137,33 @@
         <item name="android:textColor">@color/white</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <style name="ListViewV2">

```

Comment:
Maybe we should hold renaming (and doing other small code fixes) until we merge everything else? This style is used in a lot of places in the redesigned code, and I'd prefer if we avoid changing it for now.